schemas/component/readme.md

```diff
@@ -5,3 +5,19 @@ Bot Framework declarative component definitions (via .schema files which describ
 The extensions are in baseComponent.schema and bf dialog:merge -u will 
 merge baseComponent.schema + json.schema => component.schema
 
+## Published location
+
+The component JSON schemas are published at:
+
+`https://schemas.botframework.com/schemas/component/v{version}/{schema file}`
+
+Examples:
+
+`https://schemas.botframework.com/schemas/component/v1.0/component.schema`
+`https://schemas.botframework.com/schemas/component/v1.0/definitions.schema`
```

Comment:
This is obsolete and can be removed.  There is only component.schema--it includes everything in definitions.schema.  That was a point in time thing.

Reply:
Cool, I updated the readme and deleted definitions.schema from the v1.0 folder. Makes sense?

Reply:
Oh, and yes, I am planning to go and update the SDK schema files once this PR is approved and merged (the friendly URL only maps to the master branch and I need this merged first)